### PR TITLE
516 add outbox actions for get/set workitems

### PR DESCRIPTION
**Pull request by sadlerw**
* add two new gets
* create all sets
* specs are in the casejourney tests (another pr)

### CODE REVIEW

**Review by kkoskelin — APPROVED** [review]
Excellent.  I wouldn't mind if future things had names more aligned (e.g. "outbox" vs "sent") but yeah, cool.

File: web-client/src/presenter/actions/getSentWorkItemsForSectionAction.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default async ({ applicationContext }) => {
+  const user = applicationContext.getCurrentUser();
+  let sectionWorkItems = await applicationContext
+    .getUseCases()
+    .getSentWorkItemsForSection({
+      applicationContext,
+      section: user.section,
+      userId: user.userId,
```
Comment (ericsorenson) [inline web-client/src/presenter/actions/getSentWorkItemsForSectionAction.js]: Do we need the uerId here?

Comment (sadlerw) [reply web-client/src/presenter/actions/getSentWorkItemsForSectionAction.js]: nope good catch

File: web-client/src/presenter/actions/getSentWorkItemsForUserAction.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import _ from 'lodash';
+
+export default async ({ applicationContext, get }) => {
+  const useCases = applicationContext.getUseCases();
+  const userId = get(state.user.userId);
+  let workItems = await useCases.getSentWorkItemsForUser({
+    applicationContext,
+    userId,
+  });
+  workItems = _.orderBy(workItems, 'createdAt', 'desc');
```
Comment (codyseibert) [inline web-client/src/presenter/actions/getSentWorkItemsForUserAction.js:11]: I think we should do sorting in the computed, but I guess this is ok

**Pull request merged**